src/video_core/renderer_opengl/gl_shader_decompiler.cpp

```diff
@@ -2115,7 +2155,12 @@ class GLSLDecompiler final {
         UNIMPLEMENTED_IF(meta->sampler.is_array);
         const std::size_t count = operation.GetOperandsCount();
 
-        std::string expr = "texelFetch(";
+        std::string expr = "texelFetch";
+        if (!meta->aoffi.empty()) {
+            expr += "Offset";
+        }
+        expr += "(";
```

Comment:
```suggestion
        expr += '(';
```

Ditto below for a few other character constants